lib/instrument/index.js

```diff
@@ -186,7 +185,6 @@ class InstrumentService extends BaseServiceSocket {
     const reply = new DTXMessage({
       identifier: data.identifier,
       channelCode: data.channelCode,
-      selector: Buffer.alloc(EMPTY_MESSAGE_SIZE),
```

Comment:
was it redundant?

Reply:
This would change the response selector and got a exception during testmanagerd service